.github/workflows/longevity_test.yml

```diff
@@ -330,6 +336,29 @@ jobs:
             --force \
             --json \
             2>&1 | tee ${{ env.CORSO_LOG_DIR }}/maintenance_complete.txt 
+
+            # TODO(ashmrtn): We can also check that non-current versions of
+            # blobs don't have their retention extended if we want.
+            #
+            # Objects in S3 should have longer locking retention times now that
+            # we can explicitly check for.
```

Comment:
This comment may not age well.  Can you quanitify `longer`, and what happens if we go shorter than that scope?